network/tools/network-policy-enforcement-latency/go.mod

```diff
@@ -1,56 +1,113 @@
 module k8s.io/perf-tests/network/tools/network-policy-enforcement-latency
 
-go 1.22.4
+go 1.23.0
```

Comment:
changing go version could break other tests. Let's avoid doing that if not necessary